fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -176,6 +182,21 @@ public void testContinuation() {
         }
     }
 
+    @Test
+    public void testNullValue() {
```

Comment:
Since an int is not really nullable, it is hard for me to know that a null string will not break the code.

Reply:
This doesn't set the string value in one of the documents and therefore its set to null when the message is evaluated in the maintainer getFields method.

Reply:
So this tests that the maintainer doesn't crash at save time. Would it work to have a query-based test that included an `isNull` predicate against an unset field and show that this is returned?